library/kafka_info.py

```diff
@@ -58,7 +59,13 @@ def main():
     module = AnsibleModule(
         argument_spec=dict(
             resource=dict(
-                choices=['topic', 'broker', 'consumer_group', 'acl'],
+                choices=[
+                    'topic',
+                    'broker',
+                    'consumer_group',
+                    'acl',
+                    'topic-config'
```

Comment:
I was wondering if this can be merged with the current result of choice `topic`? Or maybe you saw a specific reason for adding it using a new key, like performance or API version compatibility or the fact that those are unrelated data? Should we keep a snake case format if going the `topic_config` route?

Reply:
The reason is that today "topic" resource return only partition number inside topic key, so to not break schema, i think we need to add another key, or deprecate topic resource.